.mergify.yml

```
@@ -0,0 +1,13 @@
+pull_request_rules:
+  - name: Approve and merge autobump PRs if tests are successful
+    conditions:
+      - "author=BiocondaBot"
+      - "label=autobump"
+      - "created-at>=3 days ago"
+      - "check-success=bioconda.bioconda-recipes"
```

Perhaps just checking if there is a single commit would suffice?

```suggestion
      - "check-success=bioconda.bioconda-recipes"
      - "#commits=1"
```